kokoro/builds/build.sh

```diff
@@ -0,0 +1,203 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Fail on any error.
+set -e
+
+if [ -z "${CONAN_PROFILE}" ]; then
+  CONAN_PROFILE="$(basename $(dirname "${KOKORO_JOB_NAME}"))"
+
+  # That's a temporary work-around. As long as the kokoro jobs don't carry the
+  # conan profile in its name this mapping will do the trick.
+  declare -A profile_mapping=( \
+    [gcp_ubuntu]="clang7_relwithdebinfo" \
+    [gcp_ubuntu_ggp]="ggp_relwithdebinfo" \
+    [gcp_ubuntu_test]="clang7_relwithdebinfo" \
+    [gcp_windows]="msvc2017_relwithdebinfo" \
+    [gcp_windows_test]="msvc2017_relwithdebinfo" \
+  )
+
+  CONAN_PROFILE="${profile_mapping[${CONAN_PROFILE}]-${CONAN_PROFILE}}"
+fi
+
+if [ -z "${CONAN_PROFILE}" ]; then
+  echo -n "No conan profile was set nor one could be auto-detected. "
+  echo "Aborting this build!"
+  exit 2
+fi
+
+source $(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd)/upload_symbols.sh
+
+if [ -n "$1" ]; then
+  # We are inside the docker container
+
+  readonly REPO_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../../" >/dev/null 2>&1 && pwd )"
+  readonly MOUNT_POINT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../../../../" >/dev/null 2>&1 && pwd )"
+  readonly KEYSTORE_PATH="${MOUNT_POINT}/keystore"
+
+  if [ -z "$BUILD_TYPE" ]; then
+    readonly BUILD_TYPE="$(basename "$KOKORO_JOB_NAME")"
+  fi
+
+  echo "Using conan profile ${CONAN_PROFILE} and performing a ${BUILD_TYPE} build."
+
+  set +e
+  if [ "$BUILD_TYPE" = "release" ] \
+     && ! git -C "${REPO_ROOT}" describe --tags --exact-match > /dev/null; then
+    echo -n "We are currently conducting a release build, but we aren't on a tag."
+    echo    " Aborting the build..."
+    echo -n "Maybe you missed pushing the release version tag?"
+    echo    " Please consult the release playbook for advice."
+    exit 1
+  fi
+  set -e
+
+  # That's a temporary solution. The docker containers should have the
+  # correct version of conan already preinstalled. This step will be removed
+  # when the docker containers are restructured and versioned.
+  pip3 install conan==1.27.1
```

Comment:
Should we also add `conan_package_tools==0.34.0`?

Reply:
Well, we don't need that at all currently. I want to remove that entirely at some point. And since that's just a hack I didn't bother adding it here. It would just increase the duration of the CI run (not by much though).

Reply:
ok, thanks